core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-graph.ts

```diff
@@ -105,6 +109,9 @@ export class WorkflowGraph {
     oldBreakpoint: object | undefined;
     linkID: string;
   }>();
+  public readonly operatorPortChangedSubject = new Subject<{
+    newOperator: OperatorPredicate;
```

Comment:
Should not be called `newOperator`

Reply:
The reason it was named newOperator was because since the operator's ports are changed, the operator becomes a "new" operator. However, if it makes more sense to name it something like "operator", I can change it as well.